examples/set_attitude_target/set_attitude_target.py

```diff
@@ -85,40 +84,57 @@ def arm_and_takeoff_nogps(aTargetAltitude):
         set_attitude(thrust = thrust)
         time.sleep(0.2)
 
-
-def set_attitude(roll_angle = 0.0, pitch_angle = 0.0, yaw_rate = 0.0, thrust = 0.5, duration = 0):
-    """
-    Note that from AC3.3 the message should be re-sent every second (after about 3 seconds
-    with no message the velocity will drop back to zero). In AC3.2.1 and earlier the specified
-    velocity persists until it is canceled. The code below should work on either version
-    (sending the message multiple times does not cause problems).
-    """
-    
+def send_attitude_target(roll_angle = 0.0, pitch_angle = 0.0,
+                         yaw_angle = None, yaw_rate = 0.0, use_yaw_rate = False,
```

Comment:
That is the point of lines 97-98:
```python
    if not use_yaw_rate and yaw_angle is None:
         yaw_angle = vehicle.attitude.yaw
```
so, not only the drone doesn't rotate unexpectedly, having that as a default shows the user how to circumvent this issue 😸 
This is exactly why I made this the default in the example, while still showing how to use yaw_rate.

Reply:
If the drone does rotate unexpectedly in your simulation I wouldn't know why, though, might be a bug.

Because I battle-tested this approach on the field and on a production drone.